documentation/book/getting-started.adoc

```diff
@@ -9,7 +9,9 @@
 [id='getting-started-{context}']
 = Getting started with {ProductName}
 
-{ProductName} works on all types of clusters, from public and private clouds to local deployments intended for development.
+{ProductName} works on all types of Kubernetes cluster irrespective of distribution, from public and private clouds to local deployments intended for development.
```

Comment:
Given you cannot test everything, should this say more something like _{ProductName} was designed to work on all types of Kubernetes clusters_?

Reply:
```suggestion
{ProductName} is designed to work on all types of Kubernetes cluster irrespective of distribution, from public and private clouds to local deployments intended for development.
```